apps/server/src/stores/runtimeState.ts

```diff
@@ -706,3 +700,9 @@ export function loadBlock(rundown: Rundown, state = runtimeState) {
 export function setOffsetMode(mode: OffsetMode) {
   runtimeState.runtime.offsetMode = mode;
 }
+
+function timedToPlayableIndex(metadata: RundownMetadata, index: number) {
```

Comment:
suggestion: can we add some JSDocs explaining what this does?
the function name can be confusing and is nice not having to read the implementation